pyoozie/builder.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+from __future__ import unicode_literals
+
+from pyoozie.coordinator import Coordinator
+from pyoozie.tags import Configuration, Parameters
+
+
+def _workflow_submission_xml(hadoop_user, hdfs_path, configuration=None, indent=False):
+    """Generate a Workflow XML submission message to POST to Oozie."""
+    submission = Configuration(configuration)
+    submission.update({
+        'user.name': hadoop_user,
+        'oozie.wf.application.path': hdfs_path
+    })
+    return submission.xml(indent)
+
+
+def _coordinator_submission_xml(hadoop_user, hdfs_path, configuration=None, indent=False):
+    """Generate a Coordinator XML submission message to POST to Oozie."""
+    submission = Configuration(configuration)
+    submission.update({
+        'user.name': hadoop_user,
+        'oozie.coord.application.path': hdfs_path,
+    })
+    return submission.xml(indent)
+
+
+class workflow(object):
```

Comment:
Probably because I added [`invalid-name`](http://pylint-messages.wikidot.com/messages:c0103) to pylintrc's ignore